docs/docs/troubleshooting.md

```diff
@@ -74,6 +74,16 @@ Key positions are numbered starting from `0` at the top left key on the keymap,
 A common mistake that leads to this error is to use [key press keycodes](behaviors/key-press.md) without the leading `&kp` binding. That is, having entries such as `SPACE` that should have been `&kp SPACE`.
 :::
 
+#### empty_file.c error
```

Comment:
Personally, I think it would make sense to add this to the existing `#### devicetree error` section above as another error type instead of as a new section?

e.g. add the following to the above:

A `devicetree error` followed by a reference to `empty_file.c'>  with lacks #binding-cells` indicates possible problems with improper bindings for specific codes:
```
devicetree error: <Node /soc/gpio@50000300 in '/tmp/tmp.vJq9sMwkcY/zephyr/misc/empty_file.c'> lacks #binding-cells
```
This error can be triggered by incorrect binding syntax such as `&kp BT_SEL 0` instead of `&bt BT_SEL 0`.

Reply:
I think this makes sense as well, since the line starts with `devicetree error:` just like that section.